### DISCUSSION TITLE
功能建议

### DISCUSSION THREAD

**Discussion by Zolyn** [lobehub/lobehub#3472] [Ideas]
- 移动端支持退出登录
- 服务端db模式下支持用户数据导出
- 删除s3存储桶的文件，目前删除图片只会删除数据库里的记录

**Comment by arvinxx** [comment]
> 移动端支持退出登录


@cy948 目前没做吗？

> 服务端db模式下支持用户数据导出

https://github.com/lobehub/lobe-chat/issues/3391#issuecomment-2271836133

> 删除s3存储桶的文件，目前删除图片只会删除数据库里的记录

后续会支持的

**Reply by cy948** [reply]
@arvinxx 目前确实没有主动退出的按钮。因为 NextAuth 没有profile页，只能依靠通用设置中的退出按钮，而通用设置的登陆和退出均已在 #2935 中移除。

**Reply by arvinxx** [reply]
@cy948 感觉在移动端里我的页面应该要有推出按钮的。这个应该是 mobile menu 中专有的项

**Reply by cy948** [reply]
放到 `/me` 那里吗？

**Reply by arvinxx** [reply]
对的，只有在登录后才显示退出的项
